src/iterative_telemetry/__init__.py

```diff
@@ -190,10 +190,18 @@ def send(
         impl(payload)
 
     def _send_daemon(self, payload):
-        cmd = (
-            f"import requests;requests.post('{self.url}',"
-            f"params={{'token':'{self.token}'}},json={payload})"
-        )
+        cmd = f"""
+import requests, logging
```

Comment:
will logger initialize itself this way? do we need some explicit init for it?

Reply:
Yes, but initialized the logging with latest change as well.